website/data/plugin-nav-data.json

```diff
@@ -29,6 +25,36 @@
     "title": "Debugging",
     "path": "debugging"
   },
+  {
+    "title": "Best Practices",
+    "routes": [
+      { "title": "Overview", "path": "best-practices" },
+      {
+        "title": "Handling Sensitive Data",
```

Comment:
😭 Nit: This navigation list feels a little jumbled -- I'm guessing mostly a vestige of the old list. I wonder if we should reorder it (and associated overview) to start from broadest topics to end with narrowest topics?

- Interacting with Providers
- Provider Design
- Provider Code
- Naming
- Versioning and Changelog
- Handling Sensitive Data

Reply:
Good call! done ⚡